lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent, ReactElement } from 'react';
+
+import { ThreatLevelCategory, categoryByPolicyThreatLevel } from '../../util/threatLevels';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxPolicyViolationIndicator.scss';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
```

Comment:
Minor: organizing imports

```suggestion
import React, { FunctionComponent, ReactElement } from 'react';
import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';

import { ThreatLevelCategory, categoryByPolicyThreatLevel } from '../../util/threatLevels';
import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';

import { Props, propTypes } from './types';
export { Props } from './types';

import './NxPolicyViolationIndicator.scss';
```